packages/openapi-generator/src/parser/operation.ts

```diff
@@ -47,13 +71,29 @@ export function parseParameters(
 ): OpenApiParameter[] {
   // TODO: What if this is a reference? What does OpenApi do?
   // TODO: What about oneof and other operations?
-  return (
-    operation.parameters
-      ?.map(param => resolveObject(param, refs))
-      .map(param => ({
-        ...param,
-        type: getType(resolveObject(param.schema, refs)?.type?.toString())
-      })) || []
+  return filterDuplicates(
```

Comment:
Minor: In the openapi-types the `type` and `in` attribute for the parameter are `string` type. I would prefer some type literals lik `'path' | 'query' |...` so that one has stronger typing on the potential types and locations.